src/test/unit/profile-conversion.test.js

```diff
@@ -152,14 +166,30 @@ describe('converting Google Chrome profile', function () {
       });
 
     const fs = require('fs');
-    const buffer = fs.readFileSync(
-      'src/test/fixtures/upgrades/test.chrome-unchunked.json'
+    const zlib = require('zlib');
+    const compressedBuffer = fs.readFileSync(
+      'src/test/fixtures/upgrades/test.chrome.gz'
     );
-    const text = buffer.toString('utf8');
+    const buffer = zlib.gunzipSync(compressedBuffer);
+    const events: TracingEventUnion[] = JSON.parse(buffer.toString('utf8'));
+    events.push({
```

Comment:
this data comes directly from one of the failing profiles.

Reply:
Thanks for the test!